### PR TITLE
WIP - not for merge yet - has some questions / comments within

### PR DESCRIPTION
**Pull request by amyrlam**
@serenaf Doing checks for https://github.com/ember-learn/deprecation-app/issues/69, Ember v1, see https://www.emberjs.com/deprecations/v1.x/

//

WIP / not for merge yet - sorry this isn't the cleanest commit history, more was familiarizing myself with the repo and comparing to https://www.emberjs.com/deprecations/v1.x/. I have some questions inline too which I'll try to highlight here. Figured would be best to see if this is on the right track of what you are looking for...

//

Why does the until and id at the top of the .md file (in the dotted lines) show up inline on an ember serve? Such as:
image

Seems like titles at the top of the .md file (in the dotted lines) should not have a single tick?
image

Get rid of single tick formatting in links?
image

### CODE REVIEW

**Comment by serenaf** [comment]
@amyrlam Thank you so much Amy for your help ❤️  You are totally on the right track. Can we rebase this PR from master and then we can merge it 🎉

File: content/ember/v1/access-to-instances.md [hunk 1]
```diff
@@ -48,5 +48,8 @@ App.instanceInitializer({
 
 Added in [PR #10256](https://github.com/emberjs/ember.js/pull/10256).
 
+<!-- Link isn't working? am a bit confused by repo setup?  -->
 For more information, see [the instance initializer deprecation
-guide](../instance-initializers).
+guide](/ember/instance-initializers).
```
Comment (amyrlam) [inline content/ember/v1/access-to-instances.md]: what's up with this link? if it's not in the `v1/` directory there is something unique with the routing? also will take a closer look

File: content/ember/v1/ember-reduce-computed-property.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 ---
-id: ember-reducecomputed-ember-arraycomputed
+id: ember-reducecomputedproperty-ember-arraycomputedproperty
 title: Ember.ReduceComputedProperty / Ember.ArrayComputedProperty
 until: ''
 since: '1.13'
 ---
 
+<!-- missing Ember.ReduceComputed / Ember.ArrayComputed see ember-reduce-computed.md -->
```
Comment (amyrlam) [inline content/ember/v1/ember-reduce-computed-property.md]: i set the `id` manually. but i couldn't get it to appear after `Ember.ReduceComputed / Ember.ArrayComputed`. will check if the id's are alphabetical.

what is the meaning / naming convention of the id?

**Comment by amyrlam** [comment]
thanks for the review! ok i'll rebase upstream and get this cleaned up. if there are a couple things i can't figure out, will address them separately

**Comment by amyrlam** [comment]
moved to other PR's since there was a lot of random stuff here, and will be easier to work async by addressing them separately!

File: content/ember/v1/ember-reduce-computed-property.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
 ---
-id: ember-reducecomputed-ember-arraycomputed
+id: ember-reducecomputedproperty-ember-arraycomputedproperty
```
Comment (mansona) [inline content/ember/v1/ember-reduce-computed-property.md]: Howdy folks, I don't know if we can actually change any of these ids at all. They are what is used to link to this particular deprecation from the ember-cli console.log deprecation warning. @locks can you confirm?

Comment (mansona) [reply content/ember/v1/ember-reduce-computed-property.md]: sorry! I'll comment on the other PRs 😖 

I just saw it was closed and assumed it was merged!

Comment (amyrlam) [reply content/ember/v1/ember-reduce-computed-property.md]: all good, sorry this was kind of a confusing hodgepodge. yeap let's discuss on https://github.com/ember-learn/deprecation-app/pull/87

**Pull request closed without merging**